src/navigate-pop/bookmarks-starred-items.md

```diff
@@ -0,0 +1,37 @@
+# Bookmarks & Starred Items
+
+Bookmarking and starring items makes them quickly accessible within Nautilus.
+
+---
+
+## Adding Bookmarks
+
+Adding a bookmark pins a folder below the locations listed on the left. These folders can be quickly accessed without navigating through directories.
+
+Navigate into the folder that you want to pin as a bookmark. Right-click the blank space within the folder and select `Add to Bookmarks`.
+
+![Add to Bookmarks](/images/navigate-files-folders/add-to-bookmarks.png)
+
+You can also left-lick and drag the folder over to the left panel.
+
+![Add to Bookmarks 2](/images/navigate-files-folders/add-to-bookmarks2.png)
```

Comment:
This also has your Gimp selection tool, that needs to not be there. It sounds like you're screenshotting a second time within Gimp instead of just cropping within Gimp and then saving? I would suggest exporting from Gimp properly or using some other image cropping tool so this doesn't happen anymore (I use Gwenview myself, it offers very easy crop control with click-and-drag and numbers, just press Shift-C.)

For this one in particular, I would recommend switching to a WEBM since you're showing a motion. That would also eliminate the next screenshot.

Reply:
created webm, also redid all screenshots in 22.04 (except for tiling section, I'll have to redo all those webms).